modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -535,11 +618,11 @@ export function getParticapantIndex(participant: 'user' | 'backup' | 'bitgo'): n
 }
 
 /**
- * gets particapant name by index
- * @param index particapant index
- * @returns particapant name
+ * gets participant name by index
+ * @param index participant index
+ * @returns participant name
  */
-export function getParticapantFromIndex(index: number): SignatureShareType {
+export function getParticipantFromIndex(index: number): SignatureShareType {
```

Comment:
name change needs to be breaking. see our new PR template [here](https://github.com/BitGo/BitGoJS/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and this ref [PR](https://github.com/BitGo/BitGoJS/pull/2565)

Reply:
tss ecdsa hasn't been released yet. it should be fine to break this